### PR TITLE
JI:773039 P2022-2098 [Search] Search bar can be alternately expanded and collapsed on fast tapping below the search bar field

### PR DESCRIPTION
**Pull request by krysia1**

### CODE REVIEW

**Review by pawelwojdas — COMMENTED** [review]
LGTM

File: libs/shared/src/components/search-bar/search-bar.tsx [hunk 1]
```diff
@@ -12,16 +12,20 @@ export const SearchBar: React.FC = () => {
   const [isOpen, setIsOpen] = useState(false);
   const { t } = useTranslation();
 
+  const handleFocus = () => {
```
Comment (kacperChlon) [inline libs/shared/src/components/search-bar/search-bar.tsx:15]: handleSearchBtnClick

File: libs/shared/src/components/search-bar/search-bar.tsx [hunk 2]
```diff
@@ -12,16 +12,20 @@ export const SearchBar: React.FC = () => {
...
+    setIsOpen(true);
+  };
+
   const handleClick = () => {
```
Comment (kacperChlon) [inline libs/shared/src/components/search-bar/search-bar.tsx:19]: handleSearchDrawerClick

**Review by kacperChlon — APPROVED** [review]

**Review by aniasgit — APPROVED** [review]
Besides Kacper's comments LGTM

**Pull request merged**